conda_smithy/configure_feedstock.py

```diff
@@ -1267,6 +1267,7 @@ def _azure_specific_setup(jinja_env, forge_config, forge_dir, platform):
         config_rendered = OrderedDict(
             {
                 "CONFIG": data["config_name"],
+                "SHORT_CONFIG_NAME": data["short_config_name"],
```

Comment:
Add this only if `forge_config["azure"]["store_build_artifacts"]` is true

Reply:
@isuruf added to conditional statment, thanks for the hint.